packages/expo-gl-cpp/android/dependencies.gradle

```diff
@@ -0,0 +1,341 @@
+buildscript {
+  repositories {
+    google()
+    jcenter()
+  }
+
+  dependencies {
+    classpath 'com.android.tools.build:gradle:3.5.1'
+    classpath 'de.undercouch:gradle-download-task:2.0.0'
+  }
+}
+
+import java.nio.file.Paths
+
+import de.undercouch.gradle.tasks.download.Download
+import org.apache.tools.ant.taskdefs.condition.Os
+import org.apache.tools.ant.filters.ReplaceTokens
+
+def JSC_REVISION = "236355"
+def JSC_VERSION = "${JSC_REVISION}.1.1"
+def BOOST_VERSION = "1_63_0"
+def DOUBLE_CONVERSION_VERSION = "1.1.6"
+def FOLLY_VERSION = "2018.10.22.00"
+def GLOG_VERSION = "0.3.5"
+def FLIPPER_VERSION = "0.23.4"
```

Comment:
It depends on changes, it might be a compilation error if e.g. `jsi` is using some new features or it might be linking error when launching app if sth is removed and it might be some other undefined behavior, but I don't think we need to worry about those libraries, api there should stable enough to make it very rare.

More problematic is`jsi` itself, it has already been a problem with older sdk, I needed to update jsi and rebuild jscexectutor for sdk35 and earlier so the new expo-gl would work properly. The problem manifested as a crash in runtime, based on stack trace it seems that c++ code executed a wrong function.